test/WideImage/Operation/RotateTest.php

```diff
@@ -58,7 +58,11 @@ public function testRotate45()
     {
         $img = WideImage::load(IMG_PATH . '100x100-rainbow.png');
         $new = $img->rotate(45);
-        $this->assertEquals(141, $new->getWidth());
-        $this->assertEquals(141, $new->getHeight());
+
+        // TODO: figure out why in some PHP versions this is either 140 or 141
```

Comment:
I get 143 on PHP 7.4.3 from Ubuntu and 142 on any PHP version built by Nix. Perhaps it depends on version of libgd.

Reply:
All right, I'll make this a bit less strict then to be between 140 and 143.